src/modules/flow/http-server/http-server.c

```diff
@@ -66,62 +65,28 @@ struct http_data {
     char *path;
 };
 
+struct http_server_node_type {
+    struct sol_flow_node_type base;
+    int (*post_cb)(struct http_data *mdata, struct sol_flow_node *node,
+        struct sol_http_param_value *value);
+    int (*response_cb)(struct http_data *mdata, struct sol_buffer *content, bool json);
+    int (*process_cb)(struct http_data *mdata, const struct sol_flow_packet *packet);
+    void (*send_packet)(struct http_data *mdata, struct sol_flow_node *node);
```

Comment:
I was in doubt about this name, in my head it had a different meaning yesterday night. I'll change.